debezium-testing/debezium-testing-system/pom.xml

```diff
@@ -57,10 +59,13 @@
 
     <!--Strimzi configuration-->
     <strimzi.operator.name>strimzi-cluster-operator</strimzi.operator.name>
+    <strimzi.operator.channel>stable</strimzi.operator.channel>
     <strimzi.operator.connectors>true</strimzi.operator.connectors>
     <strimzi.crd.version>v1beta2</strimzi.crd.version>
     <strimzi.kc.build>true</strimzi.kc.build>
     <strimzi.kc.log.level.dbz>DEBUG</strimzi.kc.log.level.dbz>
+    <strimzi.operator.subscription.chanel>stable</strimzi.operator.subscription.chanel>
```

Comment:
I don't really see difference between strimzi.operator.channel and strimzi.operator.subscription.channel